CHANGES.txt

```diff
@@ -1,3 +1,24 @@
+2.0.0-beta.1 (2022-12-07)
+
+Changed:
```

Comment:
CLI arguments for `planet orders request` changed as well, right?

Reply:
yeah, it's odd that got missed...